modules/gdscript/tests/scripts/analyzer/features/external_enum_as_constant.gd

```diff
@@ -0,0 +1,6 @@
+const External = preload("external_enum_as_constant_external.notest.gd")
+const MyEnum = External.MyEnum
+
+func test():
+    print(MyEnum.WAITING == 0)
+    print(MyEnum.GODOT == 1)
```

Comment:
Missing newline at EOF, so it fails style checks. (Same for the `notest` one.)

Reply:
Thanks!